### DISCUSSION TITLE
Line in Wxmaxima not high enough to show exponent of exponent [solved]

### DISCUSSION THREAD

**Discussion by Alberstein** [wxMaxima-developers/wxmaxima#1521] [Q&A]
Hi all,

when I type: in Wxmaxima:

exp(-x^2/2);

the result shown then in Wxmaxima cuts off the exponent of x, i.e. from 2 I see only the base line.

I am running on Lubuntu 20.4:

        _wxMaxima version: 19.07.0
        using wxWidgets version: wxWidgets 3.0.4
        Maxima version: 5.43.2
        Maxima build date: 2020-02-21 05:22:38
        Host type: x86_64-pc-linux-gnu
        System type: BSD BSD NIL
        Lisp implementation type: GNU Common Lisp (GCL)
        Lisp implementation version: GCL 2.6.12_

Is there anything I can do to get the whole expression shown in Wxmaxima?
I have now asked the same question at:
https://stackoverflow.com/questions/66829305/line-in-wxmaxima-not-high-enough-to-show-exponent-of-exponent
There is also a screen shot showing what I get.
Thanks
Karl

**Comment by Parlazo** [comment]
i dont know but maybe you can add it into a box command or maybe some display flags as display2d?

**Comment by gunterkoenigsmann** [comment]
In the current Nightly Build on Dautermann.at this should be resolved now...

**Reply by Alberstein** [reply]
Thanks, I took teh ubuntu build.
